templates/impala/search.html

```diff
@@ -16,7 +19,10 @@
     {{ search_form.platform }}
   {% endif %}
   <button class="search-button" type="submit" title="{{ _('Search') }}"
-         src="{{ static('img/zamboni/global/btn-search.png') }}"></button>
+         src="{{ static('img/zamboni/global/btn-search.png') }}"
+          style="text-indent:-999em;">search
```

Comment:
> I am completely lost in css files, most of them are read-only and I can't find a class that hides text, so I need to create a new one, which css file should I define the new class at ?

So I would add something like this [1] (from html5bp)  from a quick glance I don't see anything similar in our code base. This is a 'current' approach for hiding text but allowing it to be read by screen readers - you'll need to wrap the text in a span and apply a class to that for this approach though.

It looks like this file [2] is the best place for site-wide styling 

[1] https://github.com/h5bp/html5-boilerplate/blob/0c8b71eb88d9bcc8656fe3cecc76ccc3cd30c310/dist/css/main.css#L131
[2] https://github.com/mozilla/olympia/blob/master/static/css/impala/site.less